internal/datacoord/cluster.go

```diff
@@ -104,6 +104,7 @@ func (c *Cluster) Flush(ctx context.Context, nodeID int64, channel string,
 		Base: commonpbutil.NewMsgBase(
 			commonpbutil.WithMsgType(commonpb.MsgType_Flush),
 			commonpbutil.WithSourceID(Params.DataCoordCfg.GetNodeID()),
+			commonpbutil.WithTargetID(nodeID),
```

Comment:
This will be filled with the DataNode client, no need to fill it here

Reply:
It's weird that with this line removed, datanode will have lot's of node unmatched error.